homeassistant/components/filesize/__init__.py

```diff
@@ -11,13 +11,24 @@
 from .const import PLATFORMS
 
 
+def check_path(path: pathlib.Path) -> bool:
+    """Check path."""
+    exists = path.exists()
+    is_file = path.is_file()
+
+    if exists and is_file:
+        return True
+    return False
```

Comment:
```suggestion
    return path.exists() and path.is_file()
```

Reply:
Although the previous condition was `if not get_path.exists() and not get_path.is_file()`